addons/mail/models/mail_message.py

```diff
@@ -240,6 +242,28 @@ def _search_starred(self, operator, operand):
             return [('starred_partner_ids', 'in', [self.env.user.partner_id.id])]
         return [('starred_partner_ids', 'not in', [self.env.user.partner_id.id])]
 
+    @api.depends('body', 'attachment_ids', 'subtype_id', 'subtype_id.description', 'tracking_value_ids')
```

Comment:
This kind of dependency on `mail.message` might be too costly for performance, see https://github.com/odoo/odoo/pull/70436
(In my opinion due to an issue in the ORM, but if we have to live with it...)

It's likely that updating the description of a sub-type becomes impossible on a big database if I understand that other PR correctly.

I believe you have an issue with this in the test because the test unrealistically change too many things in the same env. In business code it's fine to assume is_empty won't be put in cache, then the description will change, and then is_empty would need to be read again, all in the same env.

You can resolve that either by splitting the test in smaller parts, or by flush/invalidating cache before accessing the value again between each problematic step.